signer/src/transaction_signer.rs

```diff
@@ -501,6 +519,10 @@ where
                     .await?;
             }
             WstsNetMessage::DkgPublicShares(dkg_public_shares) => {
+                tracing::info!(
+                    "handling DkgPublicShares from signer {}",
+                    dkg_public_shares.signer_id
+                );
```

Comment:
For these logs we should probably do:
```rust
                tracing::info!(
                    signer_id = %dkg_public_shares.signer_id,
                    "handling DkgPublicShares",
                );
```

Reply:
[7d7c91b](https://github.com/stacks-network/sbtc/pull/920/commits/7d7c91b65e3aed2ee28f0f1bf2b9a5a7fd106bce)